Library/Formula/faust.rb

```diff
@@ -0,0 +1,15 @@
+class Faust < Formula
+  desc "Domain-specific language for audio signal processing algorithms"
+  homepage "http://faust.grame.fr/"
+  url "https://downloads.sourceforge.net/project/faudiostream/faust-0.9.73.tgz"
+  sha256 "6512828b00e7a5a6d5ed6efc44dc54381991275abdc8aacf3a5deecd67675a74"
+
+  def install
+    system "make", "PREFIX=#{prefix}"
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    system "#{bin}/faust", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

Looks like there's a bunch of examples in this upstream PDF: http://faust.grame.fr/images/faust-tutorial2.pdf